aws_lambda_powertools/utilities/data_classes/dynamo_db_stream_event.py

```diff
@@ -182,8 +182,10 @@ def approximate_creation_date_time(self) -> Optional[int]:
         item = self.get("ApproximateCreationDateTime")
         return None if item is None else int(item)
 
+    # This override breaks the Mapping protocol of DictWrapper, it's left here for backwards compatibility with
```

Comment:
```suggestion
    # NOTE: This override breaks the Mapping protocol of DictWrapper, it's left here for backwards compatibility with
```

Capital `NOTE` helps highlight in IDE when looking for notes worthy of attention